npm upgrade does not upgrade @nrwl/eslint-plugin-nx

## Expected Behavior

Running `npm run update` should update @nrwl/* to 8.7.0

## Current Behavior
`"@nrwl/eslint-plugin-nx": "8.6.0",`
is not automatically updated and blocks the upgrade process:
```
> ng update @nrwl/workspace
Using package manager: 'npm'
Collecting installed dependencies...
Found 94 dependencies.
Fetching dependency metadata from registry...
                  Package "@nrwl/eslint-plugin-nx" has an incompatible peer dependency to "@nrwl/workspace" (requires "8.6.0", would install "8.7.0").
Incompatible peer dependencies found.
Peer dependency warnings when installing dependencies means that those dependencies might not work correctly together.
You can use the '--force' option to ignore incompatible peer dependencies and instead address these warnings later.
```

### Context

```
"@nrwl/cypress": "8.6.0",
 "@nrwl/eslint-plugin-nx": "8.6.0",
 "@nrwl/express": "8.6.0",
 "@nrwl/jest": "8.6.0",
 "@nrwl/node": "8.6.0",
 "@nrwl/workspace": "8.6.0",
```

```
Angular CLI: 9.0.0-next.11
Node: 12.9.1
OS: linux x64
Angular: 9.0.0-next.11
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.803.3
@angular-devkit/build-angular     0.803.12
@angular-devkit/build-optimizer   0.803.12
@angular-devkit/build-webpack     0.803.12
@angular-devkit/core              9.0.0-next.11
@angular-devkit/schematics        8.3.3
@angular/cdk                      9.0.0-next.0
@angular/material                 9.0.0-next.0
@ngtools/webpack                  8.3.12
@schematics/angular               8.3.3
@schematics/update                0.900.0-next.11
rxjs                              6.5.3
typescript                        3.6.4
webpack                           4.39.2
```

@jaysoo 

Running on NX 8.1.2

```console

❯ npm run update

> ult-starter-workspace@0.0.0 update /Users/ultimate-solutions/dev/ultimate-nx-starter-workspace
> ng update @nrwl/workspace

The installed Angular CLI version is older than the latest stable version.
Installing a temporary version to perform the update.
Installing packages for tooling via npm.
Installed packages for tooling via npm.
Using package manager: 'npm'
Collecting installed dependencies...
Found 107 dependencies.
Fetching dependency metadata from registry...
                  Package "@nrwl/eslint-plugin-nx" has an incompatible peer dependency to "@nrwl/workspace" (requires "8.12.1", would install "9.2.2").
✖ Migration failed: Incompatible peer dependencies found.
Peer dependency warnings when installing dependencies means that those dependencies might not work correctly together.
You can use the '--force' option to ignore incompatible peer dependencies and instead address these warnings later.
  See "/private/var/folders/6j/nbqyf2ld4wv320qvkdm71jl00000gn/T/ng-4Z1bHn/angular-errors.log" for further details.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ult-starter-workspace@0.0.0 update: `ng update @nrwl/workspace`
npm ERR! Exit status 1

``` 

NX Report
``` 
>  NX  Report complete - copy this into the issue template

  @nrwl/angular : 8.12.1
  @nrwl/cli : 8.12.1
  @nrwl/cypress : 8.12.1
  @nrwl/eslint-plugin-nx : 8.12.1
  @nrwl/express : 8.12.1
  @nrwl/jest : 8.12.1
  @nrwl/linter : 8.12.1
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : 8.12.1
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 8.12.1
  @nrwl/web : Not Found
  @nrwl/workspace : 8.12.1
  typescript : 3.7.5
```

```
     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 9.0.0-rc.12
Node: 13.12.0
OS: darwin x64

Angular: 9.0.0-rc.13
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.803.23
@angular-devkit/build-angular     0.900.0-rc.12
@angular-devkit/build-optimizer   0.900.0-rc.12
@angular-devkit/build-webpack     0.803.23
@angular-devkit/core              8.3.23
@angular-devkit/schematics        8.3.23
@angular/cdk                      9.0.0-rc.9
@angular/cli                      9.0.0-rc.12
@angular/material                 9.0.0-rc.9
@ngtools/webpack                  9.0.0-rc.12
@schematics/angular               8.3.23
@schematics/update                0.900.0-rc.12
ng-packagr                        9.0.0-rc.3
rxjs                              6.5.4
typescript                        3.7.5
webpack                           4.41.2
```


In my current projects we are not using `eslint / @nrwl/eslint-plugin-nx`, so I do not know what the status is with more updated projects.

You could close it for now.
If it occurs on future projects again, I'll create a new issue and link to this one.

--- a/packages/workspace/package.json
+++ b/packages/workspace/package.json
@@ -35,16 +35,17 @@
       "@nrwl/angular",
       "@nrwl/bazel",
       "@nrwl/cypress",
+      "@nrwl/eslint-plugin-nx",
       "@nrwl/express",
       "@nrwl/jest",
+      "@nrwl/linter",
       "@nrwl/nest",
       "@nrwl/next",
       "@nrwl/node",
       "@nrwl/nx-plugin",
       "@nrwl/react",
       "@nrwl/storybook",
-      "@nrwl/web",
-      "@nrwl/eslint-plugin-nx"
+      "@nrwl/web"
     ]
   },
   "peerDependencies": {

--- a/packages/workspace/package.json
+++ b/packages/workspace/package.json
@@ -43,7 +43,8 @@
       "@nrwl/nx-plugin",
       "@nrwl/react",
       "@nrwl/storybook",
-      "@nrwl/web"
+      "@nrwl/web",
+      "@nrwl/eslint-plugin-nx"
     ]
   },
   "peerDependencies": {

